### ISSUE TITLE
Final Report Review

### ISSUE THREAD

**Issue by posadaj** [boyangMAE/Wind-Prediction-Project#14]
The project on wind prediction is an interesting endeavor that seeks to answer a real-world problem. Ultimately the project confirmed my own inutition: the weather of recent days are more relevant in predicting tomorrow's weather than that of prevoious years. The data examined is wind speed data from the previous three years at the Ithaca, NY airport.

Again, I like that the student chose to examine a real-world problem for the course. There's a good introduction as to why this problem deserves attention. I also liked how the student generated multiple models to select from. The division of time is the most obvious and a good first step in generating multiple models. The usage of multiple loss functions is also good for finding a final model that can be used in production.

There are a few areas where the project can be improved. The report could have been presented better. The graphs are inconsistent with some containing legends while others do not and figure 3 compares two graphs that are plotted in different styles. In general, the report serves the purpose of reporting your results and methods used but not explaining your actual implementation. Thus I think some of the details could be left out in the report like the explanation of the two-day model. Simply explaining the features used will suffice, there is no need to explain the indices of the features. Also be careful when selecting a model. Performing validation on too many models can also overfit your model on the validation set. Two hundred and fifty five models seems excessive.
